scss/_typography.scss

```diff
@@ -7,15 +7,25 @@ $base-font-family: "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "
 // Base typography
 // ----------------------------------------------
 
-h1,
+h1 {
+	margin: 0;
+  padding: 0;
+  font-weight: 700;	
+}
 h2,
 h3,
 h4,
-h5,
 h6 {
   margin: 0;
```

Comment:
Margin/paddings don't need to be repeated since they'll already cascade from the initial rule.